centos_ci.sh

```diff
@@ -208,6 +216,52 @@ function docs_tar_upload() {
   echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/$LATEST."
 }
 
+function bump_to_release_version() {
+  # Set Terminal
+  export TERM=xterm-256color
+  # Add git a/c identity
+  git config user.email "admn.minishift@gmail.com"
+  git config user.name "Minishift Bot"
+  # Export GITHUB_ACCESS_TOKEN
+  export GITHUB_ACCESS_TOKEN=$GITHUB_TOKEN
+  # Create master branch as git clone in CI doesn't create it
+  git checkout -b master
+  # Bump version and commit
+  sed -i "s|MINISHIFT_VERSION = .*|MINISHIFT_VERSION = $RELEASE_VERSION|" Makefile
+  git add Makefile
+  git commit -m "cut v$RELEASE_VERSION"
+  # Push update
+  # TODO: Change to minishift once final review is done
+  git push https://$1:$GITHUB_TOKEN@github.com/$1/minishift master
+
+  if [[ "$?" = 0 ]]; then
+    echo "Minishift v$RELEASE_VERSION has been successfully released. Find the latest release here https://github.com/${1}/minishift/releases/tag/v$RELEASE_VERSION."
+  else
+    echo "Failed to release Minishift v$RELEASE_VERSION. Try to release manually."
```

Comment:
Let's send a notification to Minibot as well. See https://github.com/minishift/minishift/pull/1109/files#r130959587. In this case we have a success=false